demystifying_command_line/demystifying_command_line.md

```diff
@@ -0,0 +1,251 @@
+
+There are lots of places to learn commands.  One skill that will help is learning to effectively search the internet for a good description of the thing you're trying to do.  Chances are that you'll find someone who has already had the same challenge as you and learned how to solve it.  For example, if you're not sure how to list the files in a directory, you might search using terms like "command line what files are in my directory" or "CLI show files Windows" or something similar.  It takes practice to learn the vocabulary to create good search terms, but keep in mind that even pros who use the command line many times a day will often search for things rather than rely on their memory!  
+
+![A four-panel cartoon. People standing over a bomb with an attached keyboard yell "Rob! You use Unix! Come quick!".  The bomb display reads "To disarm the bomb, simply enter a valid `tar` command on your first try.  No googling.  You have ten seconds."  The group looks on in silence, until one person says, "Rob?".  Rob replies, "I'm so sorry."](https://imgs.xkcd.com/comics/tar.png "[Tar](https://xkcd.com/1168/) comic by xkcd, [CC BY-NC 2.5](https://xkcd.com/license.html).")
+
+One place to learn the basics of working in the command line is in our [Bash / Command Line 101](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/bash_command_line_101/bash_command_line_101.md) module.  This is a good place to do some safe, easy command line practice so you feel more confident.
+
+There are also `man` (short for *manual*) and `help` documentation pages that you can view within the command line.  For example, in a Linux-like CLI such as the Mac Terminal or Git Bash on Windows, typing `man ls` will give lots of instructions on how to use the `ls` command.
+
+<div class = "history">
+<b style="color: rgb(var(--color-highlight));">Historical context</b><br>
+
+If you try typing a `man` command, like `man ls`, chances are that there is a great deal of text, and you will need to keep hitting enter or scrolling to see all of it.  And then, you might run into the problem of not knowing how to exit the page viewer!  If you see a colon as a prompt, that's a good sign that you can type `q` to quit.  
+
+Learning how to quit out of things that are happening in the command line is a very common issue for brand new users, and it's something of a rite of passage.  
+
+There's a text editor that some people use from the command line called `vim` which is notoriously tricky to learn (we don't recommend it, we prefer `nano`).  So many people ask for help getting out of `vim` that ["How do I exit Vim?"](https://stackoverflow.com/questions/11828270/how-do-i-exit-vim) is one of [Stack Overflow's](https://www.stackoverflow.com) most frequently viewed posts! It's a question so popular that it [has its own memes](https://stackoverflow.blog/2017/05/23/stack-overflow-helping-one-million-developers-exit-vim/).  
+
+</div>
```

Comment:
```suggestion
```
Feels like this falls outside of the scope of this module, though it is good content!

Reply:
So I put this in because if people try the "man" command they might get stuck, but agree that it is a bit of a scope creep.